packages/core/src/components/toast/overlayToaster.tsx

```diff
@@ -43,6 +43,13 @@ export interface OverlayToasterState {
 
 export type OverlayToasterCreateOptions = DOMMountOptions<OverlayToasterProps>;
 
+interface OverlayToasterQueueState {
+    cancel: (() => void) | undefined;
+    isRunning: boolean;
+    toasts: ToastOptions[];
+}
+
+const QUEUE_TIMEOUT_MS = 50;
```

Comment:
What determines that the timeout should be 50ms? Is this something that we should potentially make configurable?